ymlフォルダ/VPC.yml

```
@@ -0,0 +1,145 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Network_Layer Template
+
+Resources:
+  SampleVPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.0.0.0/16
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      InstanceTenancy: default
+      Tags:
+        - Key: name
+          Value: SampleVPC
+  PublicSubnet1a:

```

論理IDにAZを明示するのも悪くはないのですが、他リージョン向けに転用しようとした場合、論理IDを書き換えなければならないので、明示しないようが使いまわしが効きます。
```suggestion
  PublicSubnet01:

```

---

AZを削除しました。